app/src/main/java/com/teamagam/gimelgimel/app/common/logging/Log4jDiskLogger.java

```diff
@@ -0,0 +1,221 @@
+package com.teamagam.gimelgimel.app.common.logging;
+
+import org.apache.log4j.Level;
+import org.apache.log4j.Logger;
+
+import java.io.File;
+
+import de.mindpipe.android.logging.log4j.LogConfigurator;
+
+
+class Log4jDiskLogger extends BaseLifecycleLogger {
+
+    private static final String VERBOSITY_USER_INTERACTION = "USER_INTERACTION";
+    private static final String VERBOSITY_LIFECYCLE = "LIFECYCLE";
+
+    private Logger mLogger;
+
+    Log4jDiskLogger(String directory, String fileName, long maxFileSize, int maxBackupLogFiles, String tag) {
+        try {
+            mLogger = createLogger(directory, fileName, maxFileSize, maxBackupLogFiles, tag);
```

Comment:
Is it OK that for every class a different log4j logger is created? or should we reuse the same log4j logger ?